data/hai/hai jobs.txt

```diff
@@ -1503,3 +1503,214 @@ mission "Hai Prisoner Roleplay"
 		payment 15000 150
 		"reputation: Hai" += 5
 		dialog "The Hai depart your ship chittering in excitement. One of them hands you your payment of <payment> before running to catch up with their friends."
+
+
+
+mission "Swiftsong Business Convention [1]"
+	name "Transport to business convention at <planet>"
+	job
+	repeat
+	description "A group of <bunks> Hai who need to get to a business convention will pay you <payment> in exchange for transporting them and their <tons> of supplies to <planet>."
+	passengers 4 2 .1
+	cargo "office supplies" 1 2
+	to offer
+		random < 10
+	source
+		government "Hai"
+		not planet "Swiftsong"
```

Comment:
Would it be better to set a minimum distance of 2? Otherwise, you might get jobs from Ya Hai. You might also get them from Icelake, which is in the same system? I'm not sure if that's intended.

Reply:
Good point but I don't really see this as a problem tbh, there's gotta be people from Ya Hai or Icelake who want to go to events on Swiftsong.